.zuul.d/playbooks/tox/main.yaml

```diff
@@ -0,0 +1,22 @@
+- hosts: all
+  tasks:
+
+    - name: Install tox
+      include_role:
+        name: ensure-tox
```

Comment:
Maybe this should be part of the pre-run so the job is more stable and there's less clutter in the console tab.

Reply:
Good idea! when you know exactly where to look you tend to forget about things like this to help reduce the clutter.

Reply:
@ianw that suggestion is the result of today's discussion in `#zuul` regarding the UI difference of Zuul vs other CIs + what folks who are used to GitHub normally expect (e.g. not seeing too much noise or log lines prefixed with Ansible-related implementation details that are usually not that important in this context).

Reply:
https://zuul.opendev.org/t/pypa/build/caaac15033494c44bcf0bc40385fa750/console